ssa.go

```diff
@@ -1268,3 +1274,14 @@ func (s Subtitles) WriteToSSA(o io.Writer) (err error) {
 	}
 	return
 }
+
+// SSAOptions
+type SSAOptions struct{}
```

Comment:
`OnUnknownSectionName` and `OnInvalidLine` need to be fields of `SSAOptions`.

You also need to create a `defaultSSAOptions func() SSAOptions` that would return `SSAOptions` with callbacks properly set for logging